install_from_github.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+
```

Comment:
Do you execute this script ?

This is re-installing rvm twice, and previous script doesn't prepare the folder opendax.